akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -193,6 +193,10 @@ object ClusterEvent {
     def withUnreachableDataCenters(unreachableDataCenters: Set[DataCenter]): CurrentClusterState =
       new CurrentClusterState(members, unreachable, seenBy, leader, roleLeaderMap, unreachableDataCenters)
 
+    /** Returns true if the address is a cluster member and that member is `MemberStatus.Up`. */
+    def isMemberUp(address: Address): Boolean =
```

Comment:
Make it `private[akka]` with all internal markers.

It's too specific to be public api since it's only for MemberUp. If we would make it public we would have to add more general purpose methods, and then it's better for users to use the `members` collection themselves.

Reply:
I wasn't sure if we would want to allow users access to it as well.